### PR TITLE
bugfix: inheritance of local profiles

### PR DESCRIPTION
**Pull request by arlimus**
This is just the start of a larger section of rework needed at this point. For now: just make it work with local inheritance again.

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.getchef.com/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## GitHub Users Who Are Not Authorized To Contribute

The following GitHub users do not appear to have signed a CLA:

* @arlimus

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

File: lib/inspec/dsl.rb [hunk 1]
```diff
@@ -6,11 +6,13 @@
...
 module Inspec::DSL
   def require_controls(id, &block)
-    ::Inspec::DSL.load_spec_files_for_profile self, id, false, &block
+    opts = { profile_id: id, include_all: true, backend: @backend, conf: @conf }
+    ::Inspec::DSL.load_spec_files_for_profile(self, opts, &block)
   end
 
   def include_controls(id, &block)
-    ::Inspec::DSL.load_spec_files_for_profile self, id, true, &block
+    opts = { profile_id: id, include_all: true, backend: @backend, conf: @conf }
```
Comment (chris-rock) [inline lib/inspec/dsl.rb]: I think that should be `include_all` false

File: lib/inspec/dsl.rb [hunk 2]
```diff
@@ -6,11 +6,13 @@
 
 module Inspec::DSL
   def require_controls(id, &block)
-    ::Inspec::DSL.load_spec_files_for_profile self, id, false, &block
+    opts = { profile_id: id, include_all: true, backend: @backend, conf: @conf }
```
Comment (chris-rock) [inline lib/inspec/dsl.rb]: I think that should be include_all false

Comment (arlimus) [inline lib/inspec/dsl.rb]: good catch, thank you!!

**Comment by chris-rock** [comment]
Thanks @arlimus one step forward :-)

**Pull request merged**